packages/dd-trace/test/plugins/versions/package.json

```diff
@@ -88,6 +88,7 @@
     "dd-trace-api": "1.0.0",
     "ejs": "3.1.10",
     "elasticsearch": "16.7.3",
+    "esbuild": "0.25.10",
```

Comment:
This is now duplicated in two package.json files. I think we should only have it in one spot.

Reply:
True, I modified the test in `index.spec.js` to use this and remove the reference from the other package.json, because this is needed to run `createSandbox`.